ofrak_core/src/ofrak/cli/ofrak_cli.py

```diff
@@ -12,11 +12,13 @@
 
 from importlib_metadata import entry_points
 
+from ofrak import __version__
```

Comment:
Totally fine to have set `__version__` for this module, but why import the dunder one instead of the original one?